files/build_templates/sonic_debian_extension.j2

```diff
@@ -668,6 +668,10 @@ fi
 SONIC_PACKAGE_MANAGER_FOLDER="/var/lib/sonic-package-manager/"
 sudo mkdir -p $FILESYSTEM_ROOT/$SONIC_PACKAGE_MANAGER_FOLDER
 j2 $BUILD_TEMPLATES/packages.json.j2 | sudo tee $FILESYSTEM_ROOT/$SONIC_PACKAGE_MANAGER_FOLDER/packages.json
+if [ "${PIPESTATUS[0]}" != "0" ]; then
```

Comment:
@stepanblyschak have you considered `shopt -s pipefail` as general solution?

Reply:
Actually no, I think it might break other things that do want to hide error?

Reply:
@stepanblyschak ok. Then no need to change.